packages/flutter_tools/lib/src/device.dart

```diff
@@ -262,6 +262,12 @@ abstract class Device {
   /// Whether it is an emulated device running on localhost.
   Future<bool> get isLocalEmulator;
 
+  /// If the device is an emulator, returns the ID shown in the output of
```

Comment:
Here and elsewhere I would add a single top line comment following dartdoc standards:

Something like:
`The unique identifier for this device, or null if it is not an emulator`

Reply:
Done!